### ISSUE TITLE
Masked DicoModel

### ISSUE THREAD

**Issue by cyriltasse** [mhardcastle/ddf-pipeline#109]
While investigating the wholes, i realised we're doing something wrong with the masked DicoModel. Specifically,

```
image_ampphase1m_masked.DicoModel
image_full_ampphase_di_m_masked.DicoModel
```

are used in `DDF` but not in `kMS`... We all agree that is an issue?

**Comment by cyriltasse** [comment]
However I thing this can't explain the holes. Maybe the presence of bad facets...

**Comment by mhardcastle** [comment]
Should be fixed by PR #113 (thanks @cyriltasse).
